### PR TITLE
added update notification on every startup

### PR DESCRIPTION
**Pull request by stefancplace**
PR to add updat-notifier to cplace-cli

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -5,7 +5,24 @@
 import * as meow from 'meow';
 import {CommandRunner} from './commands';
 import {Global} from './Global';
+import updateNotifier = require('update-notifier');
```
Comment (rose-m) [inline src/cli.ts]: Why are you using a require statement here? 🤔 In TypeScript you should use regular imports wherever possible...

File: src/cli.ts [hunk 2]
```diff
@@ -5,7 +5,24 @@
...
+import * as fs from 'fs';
+import * as path from 'path';
 
+const packageJsonContent = fs.readFileSync(path.join(__dirname, '..', 'package.json'), 'utf-8');
+const packageJson = JSON.parse(packageJsonContent);
+
+// Checks for available update on every startup
+const notifier = updateNotifier({
+                                    pkg: {
```
Comment (rose-m) [inline src/cli.ts]: Formatting?

File: src/cli.ts [hunk 3]
```diff
@@ -5,7 +5,24 @@
...
+                                        name: packageJson.name,
+                                        version: packageJson.version
+                                    },
+                                    updateCheckInterval: 0,
+                                    type: 'minor'
```
Comment (rose-m) [inline src/cli.ts]: Will this only notify on minor updates? Or also on patch updates? We should definitely notify on every version difference (including patch updates).

Comment (stefancplace) [reply src/cli.ts]: The option 'type: minor' I added is actually not valid and therefore not taken into account.
that was a mistake...

**Pull request merged**